src/frame/js/actions/config.js

```diff
@@ -0,0 +1,43 @@
+import { batchActions } from 'redux-batched-actions';
+
+import http from './http';
+import { computeColorsMetadata } from '../utils/config';
+
+
+export const SET_CONFIG = 'SET_CONFIG';
+export const RESET_CONFIG = 'RESET_CONFIG';
+
+export function setConfig(key, value) {
+    return {
+        type: SET_CONFIG,
+        key,
+        value
+    };
+}
+
+export function resetConfig() {
+    return {
+        type: RESET_CONFIG
+    };
+}
+
+export function fetchConfig() {
+    return (dispatch, getState) => {
+        const {configBaseUrl, appId} = getState().config;
+
+        return dispatch(http('GET', `/client/apps/${appId}/config`, {}, {}, configBaseUrl))
```

Comment:
they are provided by the config response (which is the call made here)... it's not provided for the config call though.